pkg/vault/test/.contract-sizes/Vault

```diff
@@ -1,2 +1,2 @@
-Bytecode	23.816
-InitCode	25.092
\ No newline at end of file
+Bytecode	23.943
```

Comment:
ooof ~120 bytes for 100 gas? Are we sure this is from this PR?

I was hoping to get some bytecode buffer to play a bit with the compiler settings; feels like we can get better gas improvements out of that.

We also have some external getters in the extension that would be better off in the main vault for the batch router.

Reply:
It seems to be from this PR, as I can influence it by changing the order. Experimenting now with shifting things around.

Reply:
It really doesn't like moving the contract addresses